From a68f5006557c18d4eb7f9b62b465104b15b83123 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Wed, 7 Jan 2015 22:06:35 -0500
Subject: [scsi] qla2xxx: Update version number to 8.07.00.08.07.1-k2

Message-id: <1420668395-3579-3-git-send-email-cdupuis@redhat.com>
Patchwork-id: 102544
O-Subject: [PATCH RHEL 7.1 2/2] qla2xxx: Update version number to 8.07.00.08.07.1-k2.
Bugzilla: 1085239
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tomas Henzl <thenzl@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Chad Dupuis <chad.dupuis@qlogic.com>

RHEL 7 specific

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_version.h b/drivers/scsi/qla2xxx/qla_version.h
index 632d0b5..a19dbfe 100644
--- a/drivers/scsi/qla2xxx/qla_version.h
+++ b/drivers/scsi/qla2xxx/qla_version.h
@@ -7,7 +7,7 @@
 /*
  * Driver version
  */
-#define QLA2XXX_VERSION      "8.07.00.08.07.1-k1"
+#define QLA2XXX_VERSION      "8.07.00.08.07.1-k2"
 
 #define QLA_DRIVER_MAJOR_VER 8
 #define QLA_DRIVER_MINOR_VER 7
-- 
1.7.1